.travis.yml

```diff
@@ -1,4 +1,14 @@
 language: scala
 sudo: false
+
 scala:
-  - 2.11.4
+  - 2.10.4
+  - 2.11.5
+
+jdk:
+  - openjdk7
+  - oraclejdk7
```

Comment:
I'd drop oraclejdk7, we should only need one v7 JDK IMHO, otherwise just a waste of build resources in the majority of cases